web/controllers/respondent_controller.ex

```diff
@@ -56,6 +56,27 @@ defmodule Ask.RespondentController do
     render(conn, "stats.json", stats: stats)
   end
 
+  defp csv_rows(csv_string) do
+    delimiters = ["\r\n", "\r", "\n"]
```

Comment:
If the allowed delimiters are 3, it would be useful to have a test for each type of delimiter.